backend/amd64/simd_selection.ml

```diff
@@ -48,6 +48,24 @@ let seq_or_avx_zeroed ~dbg seq instr ?i args =
       (Cmm_helpers.vec128 ~dbg { word0 = 0L; word1 = 0L } :: args)
   else cfg_operation (Simd.sequence seq i) args
 
+let simd_load ~mode ?i instr args =
```

Comment:
The optional argument `i` is not used yet, can we remove it or give it a name that documents what it's for,  `imm` would be better.  Similarly, `mode` is always the same so far `Indexed 0`, maybe it doesn't need to be an argument. It's better to use `Arch.identity_addressing` instead of `Indexed 0`.